msteams-platform/bots/how-to/get-teams-context.md

```diff
@@ -153,7 +155,9 @@ async def _show_members(
 
 # [JSON](#tab/json)
 
-You can directly issue a GET request on `/v3/conversations/{conversationId}/members/{userId}`, using the value of `serviceUrl` as the endpoint. The value of `serviceUrl` tends to be stable but can change. When a new message arrives, your bot should verify its stored value for `serviceUrl`.
+You can directly issue a GET request on `/v3/conversations/{conversationId}/members/{userId}`, using the value of `serviceUrl` as the endpoint. The value of `serviceUrl` tends to be stable but can change. When a new message arrives, your bot should verify its stored value for `serviceUrl`. This can be used for regualr users and anonymous users.
+
+Below is a response sample for regular user
 
 ```http
 GET /v3/conversations/19:ja0cu120i1jod12j@skype.net/members/labrown@fabrikam.com"
```

Comment:
Nit: I don't think the double quote is needed at the end :) Same for the Get Single Membmer API example for an anonymous user (below example).